src/index.html

```diff
@@ -5,8 +5,340 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#">
+        <img
+          class="logo_img"
+          src="images/logo.png"
+          alt="img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
```

Comment:
Let's make links more informative
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">Apple</a>
          </li>
```